frontend/src/app/app-routing.module.ts

```diff
@@ -11,6 +11,10 @@ const routes: Routes = [
         path: 'main',
         loadChildren: () => import('./modules/main/main.module').then((m) => m.MainModule),
     },
+    {
+        path: 'register',
+        loadChildren: () => import('./modules/sign-up/sign-up.module').then((m) => m.SingUpModule),
+    },
```

Comment:
Please consolidate your work with developer from Log In page and create one module for 2 components